app/services/govspeak_document/in_app_options.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+class GovspeakDocument::InAppOptions < GovspeakDocument::Options
+  include Rails.application.routes.url_helpers
+
+  def to_h
+    super.to_h.merge(images: in_app_images)
```

Comment:
The `to_h` here is superfluous. `super` will actually be calling `to_h` on the superclass and then we'd be calling `to_h` again on that hash.

Reply:
Oh yeah - removed :-).